wavefront_opentracing_sdk/span_context.py

```diff
@@ -37,7 +37,7 @@ def get_baggage_item(self, key):
         :return: Baggage value
         :rtype: str
         """
-        return self._baggage[key]
+        return self._baggage[key] if key in self._baggage else None
```

Comment:
Wouldn't `return self._baggage[key]` already return None if the key was not present?

Reply:
It will raise `KeyError` instead of return `None`

Reply:
`self._baggage.get(key)` should work. Though I forget if `get` works across 2.x and 3.x